docs/coordinates/skycoord.rst

```diff
@@ -304,21 +305,22 @@ dicing, and selection, using the same methods and attributes that one uses for
        [(0.,   0.), (0.,  18.), (0.,  36.), (0.,  54.), (0.,  72.)]]>
 
 Note that similarly to the `~numpy.ndarray` methods, all but ``flatten`` try to
-use new views of the data, with the data copied only if that it is impossible
-(as discussed, e.g., in the documentation for numpy :func:`~numpy.reshape`).
+use new views of the data, with the data copied only if that is impossible
+(as discussed, for example, in the documentation for NumPy
+:func:`~numpy.reshape`).
 
 
 Attributes
 ===========
 
 The |SkyCoord| object has a number of useful attributes which come in handy.
-By digging through these we'll learn a little bit about |SkyCoord| and how it
+By digging through these we will learn a little bit about |SkyCoord| and how it
 works.
 
-To begin (if you don't know already) one of the most important tools for
-learning about attributes and methods of objects is "TAB-discovery".  From
+To begin (if you do not already know), one of the most important tools for
```

Comment:
I wonder if this should just be:

```suggestion
To begin, one of the most important tools for
```

?  The "(if you do not already know)" *might* be seen by some readers as "feigning surprise"? (i.e. "what, you don't know about tab complete??")